src/main/java/dev/tricht/lunaris/item/ItemParser.java

```diff
@@ -32,6 +33,47 @@ public Item parse() {
             itemType = statsPart.getWeaponType();
         }
 
+        //If the item is a weapon, parse damage and attackspeed stats for DPS calculation
+        if (itemType instanceof WeaponItem){
```

Comment:
Should probably move this to StatsPart (or make something new) to keep the ItemParser as simple as possible

Reply:
Ill look into moving this logic into the StatsPart class